### ISSUE TITLE
Jave and scala nightly tests broken 

### ISSUE THREAD

**Issue by leezu** [apache/incubator-mxnet#17755]
```
[2020-03-03T19:53:00.983Z] [ERROR] Failed to execute goal on project mxnet-scala-demo: Could not resolve dependencies for project Demo:mxnet-scala-demo:pom:1.0-SNAPSHOT: Failed to collect dependencies at org.apache.mxnet:mxnet-full_2.11-linux-x86_64-cpu:jar:[1.7.0-SNAPSHOT,): No versions available for org.apache.mxnet:mxnet-full_2.11-linux-x86_64-cpu:jar:[1.7.0-SNAPSHOT,) within specified range -> [Help 1]
```

https://github.com/apache/incubator-mxnet/pull/17741

**Comment by zachgk** [comment]
The current issue is that the nightly maven pipeline is still publishing 1.6.0-SNAPSHOT, not the expected 1.7.0-SNAPSHOT. I added some PRs to update the version numbers for Scala (it got missed by the earlier PR) in #17764 and #17765.

As part of this, we should also move the nightly maven pipeline to target the v1.x branch instead of the master branch as well (http://jenkins.mxnet-ci.amazon-ml.com/job/restricted-publish-artifacts/). We can create a second pipeline to target master once the version numbers on master are updated to 2.0.

**Comment by szha** [comment]
@zachgk I'm working on this. it depends on #17775 (so that we don't have to introduce additional jenkins global variable...)

**Comment by zachgk** [comment]
@szha The scala CD is not part of the CD system that you are working on located at https://github.com/apache/incubator-mxnet/tree/master/cd, but a completely different one located at https://github.com/apache/incubator-mxnet/tree/master/ci/publish. It also has a separate Jenkins pipeline (http://jenkins.mxnet-ci.amazon-ml.com/job/restricted-publish-artifacts/). There shouldn't be any dependency on your work to get this issue resolved

**Comment by zachgk** [comment]
The old PR was merged and I updated the pipeline to also build the `v1.x` branch and started the [first build](http://jenkins.mxnet-ci.amazon-ml.com/job/restricted-publish-artifacts/job/v1.x/). Once this build passes, this issue should be resolved.

**Comment by zachgk** [comment]
The build passed and the nightly tests are now [passing](http://jenkins.mxnet-ci.amazon-ml.com/job/NightlyTests/job/master/621/).
